csharp/src/Ice/Communicator.cs

```diff
@@ -2,8 +2,8 @@
 
 #nullable enable
 
-using Ice.Internal;
 using System.Net.Security;
+using Ice.Internal;
 
```

Comment:
Why are system directives moved up, if we have `dotnet_sort_system_directives_first=false:warning`?